data/mil2525d/utilities/style-utilities/merge-stylx-utilities/Readme.md

```diff
@@ -3,38 +3,53 @@
 ## Purpose
 
 * Merges:
-* A version of the .stylx that is edited and maintain in ArcGIS Pro
+    * A version of the .stylx that is edited and maintain in ArcGIS Pro
```

Comment:
Addressed in https://github.com/csmoore/military-features-data/commit/06bcb08034f2fdfb7c2d7d759dd69f975ea38fc3